### DISCUSSION TITLE
Watched Movie "DB" - is this an appropriate exercise for Bridgetown?

### DISCUSSION THREAD

**Discussion by philiprhoades** [bridgetownrb/bridgetown#590] [General]
People,

I want to build a page that is constructed from a CSV file in _data (eg) with fields like:

date
title
duration
plot
notes

and produce something like what Rails does for viewing the table - is that possible / sensible in Bridgetown?

Thanks,
Phil.

**Comment by ayushn21** [comment]
I'm afraid I don't understand what you mean by: _produce something like what Rails does for viewing the table_.

Could you please elaborate? At first glance it does sound like something Bridgetown is ideal for, but I'd like to understand your use case a bit further.

**Comment by philiprhoades** [comment]
Thanks for responding - I will try to clarify:

Well, in the versions of Rails prior to v7, if you used the built-in scaffolding facility to make a quick and minimal table, then after populating the table with some records, the default view of the table just looked like a spreadsheet view of records with fields.  This scaffolding view function was lost with v7 but I think for what I am thinking of, even an sqlite DB is overkill and I should just be able to use a CSV file from _data to create one long table page view . . 

Of course there would be a few nice enhancements like being able to search and sort by fields etc.  The main thing I would need to do after the table view creation stage is to click on a record and pop up a window with the Plot and Notes in it.

Does that make sense?

**Reply by ayushn21** [reply]
Yeah that makes total sense. Bridgetown is perfect for that! 

Place your CSV file in the `_data` folder like you mentioned and it'll be available in the HTML files to iterate over. Jared just shared this gem on the Bridgetown Discord yesterday which might make your life even easier:

https://github.com/jeremyevans/thamble

If you run into any problems, feel free to post here or in the Discord and someone will give you a hand.
